charts/llm-engine/templates/service_template_config_map.yaml

```diff
@@ -443,7 +443,7 @@ data:
     data:
       raw_data: ${ENDPOINT_CONFIG_SERIALIZED}
   horizontal-pod-autoscaler.yaml: |-
-    apiVersion: autoscaling/v2beta2
+    apiVersion: ${API_VERSION}
```

Comment:
Hmm a little odd to call this `API_VERSION` - actually I guess it's fine, assuming that `API_VERSION` is just an env var that's contextualized to this file. For instance, I wouldn't expect a global helm value called `ApiVersion` to be appropriate for something specific like an HPA API version.

Reply:
This value contextualized to just this file.